### PR TITLE
[LC-1007] Remove or Replace lines in BlockChain, BlockManager

### PR DESCRIPTION
**Pull request by metalg0su**

### CODE REVIEW

File: loopchain/channel/channel_inner_service.py [hunk 1]
```diff
@@ -613,7 +614,7 @@ def get_tx_info(self, tx_hash):
             return message_code.Response.success, tx_info
         else:
             try:
-                return message_code.Response.success, self._block_manager.get_tx_info(tx_hash)
+                return message_code.Response.success, self._blockchain.find_tx_info(tx_hash)
             except KeyError as e:
                 logging.error(f"get_tx_info error : tx_hash({tx_hash}) not found error({e})")
```
Comment (hyerim-kim) [inline loopchain/channel/channel_inner_service.py:619]: It should be better if the below log is updated with the edited method name.

Comment (metalg0su) [reply loopchain/channel/channel_inner_service.py]: Please check that the log line is changed as you intended.

File: loopchain/blockchain/blockchain.py [hunk 2]
```diff
@@ -784,10 +784,6 @@ def prevent_next_block_mismatch(self, next_height: int) -> bool:
                                f"loopchain({next_height})/score({score_last_block_height})")
             return True
 
-    @staticmethod
-    def __get_tx_list_key(address, index):
-        return conf.TX_LIST_ADDRESS_PREFIX + (address + str(index)).encode(encoding=conf.HASH_KEY_ENCODING)
```
Comment (metalg0su) [inline loopchain/blockchain/blockchain.py:789]: Check usage `TX_LIST_ADDRESS_PREFIX`

Comment (metalg0su) [reply loopchain/blockchain/blockchain.py:789]: Removed at `loopchain/configure_default.py`

**Pull request merged**